cht/adapter.py

```diff
@@ -2,307 +2,296 @@
 import sys
 import numpy as np
 import numpy.linalg
-from mpi4py import MPI
-from Cata.cata import *
-from Utilitai.partition import *
+#from mpi4py import MPI
+from code_aster.Cata.Commands import *
+from Utilitai import partition
+import precice
 
 np.set_printoptions(threshold=np.inf)
 
-precice_root = os.getenv('PRECICE_ROOT')
-precice_python_adapter_root = precice_root+"/src/precice/adapters/python"
-sys.path.insert(0, precice_python_adapter_root)
-
-import PySolverInterface
-from PySolverInterface import *
-
 class Adapter:
 
-    def __init__(self, preciceConfigFile, participantName, config, MESH, MODEL, MAT, isNonLinear=False):
-        self.interfaces = []
-        self.numInterfaces = len(config)
-        self.MESH = MESH
-        self.MODEL = MODEL
-        self.MAT = MAT
-        self.LOADS = []
-        self.isNonLinear = isNonLinear
-        self.participantName = participantName
-        self.preciceDt = -1
-        self.precice = PySolverInterface(participantName, 0, 1)
-        self.precice.configure(preciceConfigFile)
-        self.configure(config)
-    
-    def configure(self, config):
-        L = [None] * self.numInterfaces        # Loads
-        SM = [None] * self.numInterfaces    # Shifted meshes
-        for i in range(self.numInterfaces):
-            # Shifted mesh (interface nodes displaced by a distance delta in the direction of the surface normal
```

Comment:
Why removing this comment?

Reply:
These comments seemed unnecessary to me. If you think they're helpful, I will reinsert them.